package.json

```diff
@@ -31,28 +31,30 @@
     "zone.js": "~0.14.6"
   },
   "devDependencies": {
-    "@angular-builders/jest": "17.0.3",
-    "@angular-devkit/build-angular": "^17.3.5",
-    "@angular-eslint/builder": "17.3.0",
-    "@angular-eslint/eslint-plugin": "17.3.0",
-    "@angular-eslint/eslint-plugin-template": "17.3.0",
-    "@angular-eslint/schematics": "17.3.0",
-    "@angular-eslint/template-parser": "17.3.0",
-    "@angular/animations": "^17.3.5",
-    "@angular/cli": "^17.3.5",
-    "@angular/compiler": "^17.3.5",
-    "@angular/compiler-cli": "^17.3.5",
-    "@angular/language-service": "^17.3.5",
-    "@angular/router": "^17.3.5",
+    "@angular-builders/jest": "^18.0.0-beta.3",
+    "@angular-devkit/build-angular": "^18.0.2",
+    "@angular-eslint/builder": "17.5.1",
```

Comment:
do these have 18.x versions or are they not tied to angular? they look like they are

Reply:
Updated. 
These were not picked up by `ng update`.